src/ocm/components/clusterWizard/ClusterWizardFooter.tsx

```diff
@@ -45,6 +47,20 @@ const ValidationSection = ({
           {_alertContent}
         </Alert>
       )}
+      {currentStepId === 'host-discovery' &&
```

Comment:
I think we are now doing too much logic in this component.
Creating a separate component to handle when to show it would be better IMO.

I mean something like

```
{cluster && currentStepId === 'host-discovery' && (
  <ClusterPlatformIntegrationHint cluster={cluster} />
)}
```

Reply:
Ok. I'll try to do a separate component